helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -509,10 +510,15 @@ public void rollingUpdate(final DeploymentGroup deploymentGroup,
       final RollingUpdateOp op = getInitRollingUpdateOps(updated, MANUAL, client);
       operations.addAll(op.operations());
 
+      log.info("starting zookeeper transaction for rolling-update on "
+               + "deployment-group name={} job={}. List of operations: {}",
+          deploymentGroup.getName(), jobId, operations);
+
       client.transaction(operations);
 
       emitEvents(DEPLOYMENT_GROUP_EVENTS_KAFKA_TOPIC, op.events());
-      log.info("finished rolling-update on deployment-group: name={}", deploymentGroup.getName());
+      log.info("finished rolling-update on deployment-group: name={}, job={}",
```

Comment:
Seems a bit misleading to say that the rolling-update finished here. I'd interpret "rolling-update finished" as "my job rolled out to all hosts".